examples/chip-tool/commands/clusters/ModelCommand.cpp

```diff
@@ -35,11 +35,20 @@ CHIP_ERROR ModelCommand::RunCommand()
         return SendGroupCommand(GroupIdFromNodeId(mNodeId), fabricIndex);
     }
 
-    ChipLogProgress(chipTool, "Sending command to node 0x%" PRIx64, mNodeId);
-    return CurrentCommissioner().GetConnectedDevice(mNodeId, &mOnDeviceConnectedCallback, &mOnDeviceConnectionFailureCallback);
+    if (mViaPase.ValueOr(0) == 1)
+    {
+        ChipLogProgress(chipTool, "Sending command via PASE to node 0x%" PRIx64, mNodeId);
+        CommissioneeDeviceProxy * device = CurrentCommissioner().FindCommissioneeDevice(mNodeId);
```

Comment:
This can return nullptr if you haven't opened a pase connection, so that should be checked before sending the command.

Reply:
Checked now and using the public API.